adam-cli/src/main/scala/org/bdgenomics/adam/cli/Transform.scala

```diff
@@ -115,6 +115,8 @@ class TransformArgs extends Args4jBase with ADAMSaveAnyArgs with ParquetArgs {
   var mdTagsFragmentSize: Long = 1000000L
   @Args4jOption(required = false, name = "-md_tag_overwrite", usage = "When adding MD tags to reads, overwrite existing incorrect tags.")
   var mdTagsOverwrite: Boolean = false
+  @Args4jOption(required = false, name = "-bin_quality_scores", usage = "Rewrites quality scores of reads into bins. User must provide bin description string.")
```

Comment:
How about
Rewrites quality scores of reads into bins from a string of bin descriptions, e.g. 0,20,10;20,40,30.